### ISSUE TITLE
Para 1.42.2 doesn't start

### ISSUE THREAD

**Issue by mzch** [Erudika/para#91]
Para `1.42.2` doesn't start.

```
Dec 20 08:47:17 n16 systemd[1]: Started Para.
Dec 20 08:47:18 n16 java[10256]: Exception in thread "main" java.lang.ClassNotFoundException: com.erudika.para.Run
Dec 20 08:47:18 n16 java[10256]: #011at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
Dec 20 08:47:18 n16 java[10256]: #011at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
Dec 20 08:47:18 n16 java[10256]: #011at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151)
Dec 20 08:47:18 n16 java[10256]: #011at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
Dec 20 08:47:18 n16 java[10256]: #011at java.base/java.lang.Class.forName0(Native Method)
Dec 20 08:47:18 n16 java[10256]: #011at java.base/java.lang.Class.forName(Class.java:398)
Dec 20 08:47:18 n16 java[10256]: #011at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:46)
Dec 20 08:47:18 n16 java[10256]: #011at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
Dec 20 08:47:18 n16 java[10256]: #011at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
Dec 20 08:47:18 n16 java[10256]: #011at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:467)
Dec 20 08:47:18 n16 systemd[1]: para.service: Main process exited, code=exited, status=1/FAILURE
Dec 20 08:47:18 n16 systemd[1]: para.service: Failed with result 'exit-code'.
```
- java -version
```
openjdk version "11.0.13" 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8-post-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 11.0.13+8-post-Debian-1deb11u1, mixed mode, sharing)
```

**Comment by albogdano** [comment]
This is fixed in `1.43.0`. **WARNING**: Be very careful when upgrading to `1.43.0` - there are some breaking changes like refactoring, H2 has been upgraded and your database needs to be migrated, Lucene is upgraded to v9 and you need to do a full reindex.

**Comment by albogdano** [comment]
See release notes here: https://github.com/Erudika/para/releases/tag/v1.43.0

**Comment by mzch** [comment]
Not fixed in `1.43.0`. It doesn't start. I use MongoDB as database using `para-dao-mongodb-1.36.0-shaded.jar`.

```
Dec 20 09:42:25 n16 systemd[1]: Started Para.
Dec 20 09:42:26 n16 java[11055]: Exception in thread "main" java.lang.ClassNotFoundException: com.erudika.para.Run
Dec 20 09:42:26 n16 java[11055]: #011at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
Dec 20 09:42:26 n16 java[11055]: #011at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
Dec 20 09:42:26 n16 java[11055]: #011at org`.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151)
Dec 20 09:42:26 n16 java[11055]: #011at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
Dec 20 09:42:26 n16 java[11055]: #011at java.base/java.lang.Class.forName0(Native Method)
Dec 20 09:42:26 n16 java[11055]: #011at java.base/java.lang.Class.forName(Class.java:398)
Dec 20 09:42:26 n16 java[11055]: #011at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:46)
Dec 20 09:42:26 n16 java[11055]: #011at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
Dec 20 09:42:26 n16 java[11055]: #011at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
Dec 20 09:42:26 n16 java[11055]: #011at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:467)
Dec 20 09:42:26 n16 systemd[1]: para.service: Main process exited, code=exited, status=1/FAILURE
Dec 20 09:42:26 n16 systemd[1]: para.service: Failed with result 'exit-code'.
Dec 20 09:42:26 n16 systemd[1]: para.service: Consumed 1.122s CPU time.
```

**Comment by albogdano** [comment]
Sorry about that! I realized what was the problem and released `1.43.1` just now.
